src/encoded/ingestion/ingestion_processors.py

```diff
@@ -0,0 +1,42 @@
+from contextlib import contextmanager
+from typing import Generator, Union
+from dcicutils.sheet_utils import load_items
+from snovault.ingestion.ingestion_processors import ingestion_processor
+from snovault.types.ingestion import SubmissionFolio
+from .data_validation import summarize_validate_data_problems, validate_data_against_schemas
+from .loadxl_extensions import load_data_into_database, summarize_load_data_into_database_response
+from .submission_folio import SmahtSubmissionFolio
+
+
+def includeme(config):
+    config.scan(__name__)
+
+
+@ingestion_processor("metadata_bundle")
+@ingestion_processor("family_history")  # TODO: Do we need this?
+def handle_metadata_bundle(submission: SubmissionFolio) -> None:
+    with submission.processing_context():
+        _process_submission(SmahtSubmissionFolio(submission))
+
+
+def _process_submission(submission: SmahtSubmissionFolio) -> None:
+    with _load_data(submission) as data:
+        validate_data_problems = validate_data_against_schemas(data, portal_vapp=submission.portal_vapp)
+        if validate_data_problems:
+            validate_data_summary = summarize_validate_data_problems(validate_data_problems, submission)
+            submission.record_results(validate_data_problems, validate_data_summary)
+            # If there are data validation problems then trigger an exception so that a traceback.txt
+            # file gets written to the S3 ingestion submission bucket to indicate that there is an
+            # error; this is done in snovault.types.ingestion.SubmissionFolio.processing_context.
+            # TODO: I *think* this is the right thing to do; it is an exceptional
+            # situation but we just happened to have caught it programmatically.
+            raise Exception(validate_data_problems)
```

Comment:
So long as this results in `errors` being appropriately populated on the ingestion submission, sounds good